src/plugins/persistence/couch/CouchObjectProvider.js

```diff
@@ -102,6 +104,89 @@ export default class CouchObjectProvider {
         return this.request(identifier.key, "GET").then(this.getModel.bind(this));
     }
 
+    abortGetChanges(identifier) {
+        const controller = this.controllers[identifier.key];
+        if (controller) {
+            controller.abort();
+            this.controllers[identifier.key] = undefined;
+        }
+
+        return true;
+    }
+
+    async getChanges(identifier, options) {
```

Comment:
I'd like to rename this to `observeObjectChanges`. See general comments, I'd like to make this method focused on just sitting on a response and responding to changes.

Reply:
Done